export.py

```diff
@@ -184,6 +184,15 @@ def open_transactions_page(br):
 
     return br
 
+
+@messages('Opening statements page...', 'OK', 'Exiting...')
+def open_statements_page(br):
+    br.set_handle_robots(False)
+    br.open('https://28degrees-online.gemoney.com.au/wps/myportal/ge28degrees/public/statements/viewstatements/')
```

Comment:
I know I am guilty of this crime myself (see https://github.com/bedecarroll/28degrees-export/blob/statements/export.py#L130) but I think we should start auto-detecting the URLs such as the ones for Transactions and Statements page and remove hardcoding. Will log a ticket for that.